locales/it/faq/faq25.md

```diff
@@ -1 +1,6 @@
-I metodi di riconoscimento basati sulla biometria si sono rivelati nel tempo più sicuri delle password numeriche o alfa-numeriche, spesso vulnerabili o involontariamente comunicate ad altre persone. Il riconoscimento biometrico ti offre il vantaggio di velocizzare l'autenticazione, rispetto all'inserimento del Codice di sblocco, e ti permette di svolgere più velocemente le operazioni all'interno dell'app IO.
\ No newline at end of file
+No, i messaggi ricevuti non possono essere cancellati dall'app.
+
+Per questo abbiamo creato la sezione ‘*Archiviati*’, dove puoi spostare i messaggi
```

Comment:
Thank you @bfabio 
Could you remove line breaks in the second block?

Reply:
@Undermaken Done :+1: